index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://unpkg.com/flickity@2/dist/flickity.pkgd.min.js"></script>
+
+  <title>
+      San's Portfolio
+  </title>
+</head>
+<body>
+
+    <section id="home">
+    <header>
+      <a href="#" class="logo">SAN ROBINSON</a>
+      <ul id="menu"> 
+          <li><a href="#home" class="active">home</a></li>
+          <li><a href="#about">about me</a></li>
+          <li><a href="#portfolio">porfolio</a></li>
+          <li><a href="SanRSWEResume.html">resume</a></li>
```

Comment:
You can add `target="_blank"` to your a tag and it will open your resume in a new tab so the user can stay on your website.